src/main/java/io/r2dbc/postgresql/PostgresqlServerErrorException.java

```diff
@@ -166,80 +167,78 @@ public String getCode() {
      *
      * @return the value of the {@link FieldType#COLUMN_NAME} field
      */
-    public Optional<String> getColumnName() {
```

Comment:
I'd prefer not to remove `Optional` in this error.  It's method signatures are accurate, it's not in a hot code path, and given how the PostgreSQL error is such a bag of mostly empty fields, I think pointing out the mostly emptiness is worthwhile. @mp911de thoughts?

Reply:
Exception use is fine. If an exception becomes the hot code path, then I guess we have a different kind of problem then.